tests/sentry/api/endpoints/test_broadcast_index.py

```diff
@@ -60,6 +62,24 @@ def test_basic_user_with_all(self):
         assert len(response.data) == 1
         assert response.data[0]["id"] == str(broadcast1.id)
 
+    def test_organization_filtering(self):
+        org = self.create_organization()
+        self.create_member(organization=org, user=self.user, role="admin")
```

Comment:
nit: use `self.organization` in tests
```suggestion
        self.create_member(organization=self.organization, user=self.user, role="admin")
```